frontend/src/Components/RegistrationQuestionsElement.tsx

```diff
@@ -0,0 +1,38 @@
+export default function RegistrationQuestionsElement(){
+    return <div>
+        {/*<h1 th:text="#{label.form.title}">form</h1>*/}
```

Comment:
will be needed in the future?

Reply:
It was needed in the older Versions, but now it  can be trashed, I will remove it, thanks for the hint :-)